cirq/google/engine/engine.py

```diff
@@ -16,8 +16,8 @@
 
 As an example, to run a circuit against the xmon simulator on the cloud,
     engine = cirq.google.Engine(project_id='my-project-id')
-    options = cirq.google.JobConfig(program_id='hello-world')
-    result = engine.run(options, circuit, repetitions=10)
+    program = engine.create_program(ciruit, 'my-experiment')
+    result = engine.run(program=program, repetitions=10)
```

Comment:
ah I was thinking we would use a different pattern 

```
engine = cirq.google.Engine(project_id='my-project-id')
program = engine.create_program(circuit, 'my-experiment')
program.run_sweeps(job_config=job_config, params=params0, repetitions=10, processor_ids=['gmon'])
program.run_sweeps(job_config=job_config, params=params1, repetitions=10, processor_ids=['gmon'])
...
```

Reply:
Updated to reflect this while keeping old Engine run() and run_sweep() methods. PTAL.